### PR TITLE
166703218 startup config

### PR DESCRIPTION
**Pull request by craigastuckey**
initial setup of Startup.cs and appsettings.json

### CODE REVIEW

**Comment by escalonn** [comment]
wait to request my review until sadiki has approved

File: src/Housing.Profile.Service/appsettings.json [hunk 1]
```diff
@@ -1,8 +1,60 @@
...
-    "LogLevel": {
-      "Default": "Warning"
+  "ConnectionStrings": {
+    "DbInDockerStack": "mongodb://db",
+    "DbInDockerMachineContainer": "mongodb://192.168.99.100"
+  },
+  "Settings": {
+    "Queues": false,
+    "DbConnection": "DbInDockerStack",
+    "ServiceHubConnection": "DockerNetworkHttp"
```
[the remark this answers is not in the document]
Comment (craigastuckey) [reply src/Housing.Profile.Service/appsettings.json]: fixed

File: src/Housing.Profile.Service/appsettings.json [hunk 2]
```diff
@@ -1,8 +1,60 @@
...
+  },
+  "Queues": {
+    "QueueSenderName": "selection-to-users",
+    "QueueReceiverName": "to-selections"
```
[the remark this answers is not in the document]
Comment (craigastuckey) [reply src/Housing.Profile.Service/appsettings.json]: fixed

File: src/Housing.Profile.Service/Startup.cs [hunk 3]
```diff
@@ -18,14 +21,36 @@ public class Startup
         public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
+            Log.Logger = new LoggerConfiguration()
+                .Enrich.FromLogContext()
+                .WriteTo.Console()
+                .CreateLogger();
```
[the remark this answers is not in the document]
Comment (craigastuckey) [reply src/Housing.Profile.Service/Startup.cs]: fixed

File: src/Housing.Profile.Service/Housing.Profile.Service.csproj [hunk 4]
```diff
@@ -9,8 +9,15 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
```
Comment (escalonn) [inline src/Housing.Profile.Service/Housing.Profile.Service.csproj]: the `Microsoft.AspNetCore.App` metapackage includes this one already; we can get version conflicts if we also include its dependencies with specific version numbers. so just remove it

File: src/Housing.Profile.Testing/Housing.Profile.Testing.csproj [hunk 5]
```diff
@@ -11,13 +11,20 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.2.0" />
```
Comment (escalonn) [inline src/Housing.Profile.Testing/Housing.Profile.Testing.csproj]: these two are also in the metapackage already.

File: src/Housing.Profile.Testing/Housing.Profile.Testing.csproj [hunk 6]
```diff
@@ -11,13 +11,20 @@
...
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Moq" Version="4.11.0" />
+    <PackageReference Include="Serilog" Version="2.8.0" />
```
Comment (escalonn) [inline src/Housing.Profile.Testing/Housing.Profile.Testing.csproj]: do we really need serilog in the tests?

File: src/Housing.Profile.Service/Housing.Profile.Service.csproj (hunk 4 above)
Comment (craigastuckey) [reply src/Housing.Profile.Service/Housing.Profile.Service.csproj]: This was a fix to an error I kept getting about VS not being able to find the package's assembly. This is where I found the fix.
https://stackoverflow.com/a/50401153

File: src/Housing.Profile.Testing/Housing.Profile.Testing.csproj (hunk 5 above)
Comment (craigastuckey) [reply src/Housing.Profile.Testing/Housing.Profile.Testing.csproj]: fixed

File: src/Housing.Profile.Testing/Housing.Profile.Testing.csproj (hunk 6 above)
Comment (craigastuckey) [reply src/Housing.Profile.Testing/Housing.Profile.Testing.csproj]: fixed

**Pull request merged**